umpv - force playback of the first added file

andreym333 opened the pull request:
If mpv is already running, the files passed to umpv are appended
to mpv's internal playlist and the first file added starts playing.
Old playlist entries are retained.

andreym333 committed:
umpv - force playback of the first added file

If mpv is already running, the files passed to umpv are appended
to mpv's internal playlist and the first file added starts playing.
Old playlist entries are retained.

sfan5 commented:
I can see append-play being useful if you have idle enabled, but I'd say that stopping the current playback (`playlist-play-index none`) is an unwelcome behavior chance for many umpv users.

andreym333 committed:
umpv: add "--skip-old-entries" option

If option "-s" or "--skip-old-entries" is specified, mpv will skip
old playlist entries and start playback from the first new playlist
entry. Otherwise, playback of the current file continues.

andreym333 commented:
I added the "--skip-old-entries" option which will activate the behavior that I need.

jimman2003 commented:
@andreym333 please sync your as to obtain my changes.. 
:)

andreym333 committed:
Merge remote-tracking branch 'upstream/master' into umpv-force-playback

jimman2003 reviewed (approved):
Well done!

Dudemanguy commented:
Closing some very old PRs that seemed to stall out. Feel free to complain if you intend to work on it.
